qcodes/instrument_drivers/Keysight/keysight_b220x.py

```diff
@@ -0,0 +1,181 @@
+from qcodes import VisaInstrument
+from qcodes.utils.validators import MultiType, Ints, Enum, Lists
+
```

Comment:
> Note: when switching from free to single mode, 1-to-multiple connections will not be changed until the connection is changed!!

Does it make sense to implement such a warning in the driver? Or implement that the change of the 1-to-multiple connections?

I assume it's not an instrument bug, but a feature, right?))

Reply:
Let's say its an undocumented feature of the instrument 😜.

I will add a warning, but I will not implement a check or automatic release of the multiple connections (for similar reasons I wrote above in https://github.com/QCoDeS/Qcodes/pull/1373#discussion_r232925963)

Reply:
ok, perfect!

Reply:
Added a warning `When going from *free* to *single* mode existing connections are not released.`